pinot-spi/src/main/java/org/apache/pinot/spi/config/table/assignment/InstanceAssignmentConfig.java

```diff
@@ -28,29 +28,37 @@
 
 public class InstanceAssignmentConfig extends BaseJsonConfig {
 
+  @JsonPropertyDescription("Configuration for the instance assignment strategy")
```

Comment:
The description is not super clear / explanatory. Suggest including a bit more detail